torch/nn/init.py

```diff
@@ -154,7 +154,14 @@ def normal_(tensor: Tensor, mean: float = 0., std: float = 1.) -> Tensor:
         return torch.overrides.handle_torch_function(normal_, (tensor,), tensor=tensor, mean=mean, std=std)
     return _no_grad_normal_(tensor, mean, std)
 
-def trunc_normal_(tensor: Tensor, mean: float = 0., std: float = 1., a: float = -2., b: float = 2.) -> Tensor:
+def trunc_normal_(
+    tensor: Tensor,
+    mean: float = 0.,
+    std: float = 1.,
+    a: float = -2.,
+    b: float = 2.,
+    generator: torch.Generator = None
```

Comment:
```suggestion
    generator: Optional[torch.Generator] = None
```
Better explicit than implicit optional. You'll also need `from typing import Optional` at the top of the file.

Reply:
Sounds great, about to fix -- luckily, it looks like there aren't any JIT issues thus far. They may arise from future commits to the rest of the init functions (hopefully done by tomorrow excluding docs/testing), but that's a nice sign for now.